edge-modules/api-proxy-module/docker/linux/amd64/Dockerfile

```diff
@@ -7,8 +7,8 @@ ARG DIR=.
 
 FROM alpine:3.12.0
 WORKDIR /app
-COPY target/x86_64-unknown-linux-musl/release/api-proxy-module .
-COPY templates .
+COPY ./docker/linux/amd64/api-proxy-module .
+COPY ./docker/linux/amd64/templates .
```

Comment:
what is templates folder?

Reply:
btw, would you please add comments to describe EXPOSE ports are used for what.  E.g. 5000 is default port for docker container registry.  I have another comment before, ask what happen if customer use different port for their local container registry?

Reply:
./docker/linux/amd64/api-proxy-module is the generated executable.
./docker/linux/amd64/templates is the default configuration for API proxy.

Reply:
Fixed!

Reply:
what is fixed? You mentioned fixing the name from "template" to "conf"/"config" will involve many files.  I agreed this is not a big deal.